### PR TITLE
br: golang lint and 2nd restore progress file does not created

### PR DESCRIPTION
**Pull request by fengou1**
### What problem does this PR solve?
1. Golang lint2. 
2. 2nd restore progress file does  not created 
Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/task/restore_data.go [hunk 1]
```diff
@@ -177,9 +175,10 @@ func RunResolveKvData(c context.Context, g glue.Glue, cmdName string, cfg *Resto
 		return errors.Trace(err)
 	}
 
-	// progress = read meta + send recovery + resolve kv data.
-	progress := g.StartProgress(ctx, cmdName, int64(numOnlineStore*3), !cfg.LogProgress)
-	go progressFileWriterRoutine(ctx, progress, int64(numOnlineStore*3), cfg.ProgressFile)
+	log.Debug("total tikv", zap.Int("total", numBackupStore), zap.Int64("int", int64(numBackupStore)), zap.String("progress file", cfg.ProgressFile))
```
Comment (D3Hunter) [inline br/pkg/task/restore_data.go:178]:
```suggestion
	log.Debug("total tikv", zap.Int("total", numBackupStore), zap.String("progress file", cfg.ProgressFile))
```

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

**Review by D3Hunter — APPROVED** [review]

**Review by 3pointer — APPROVED** [review]

**Comment by 3pointer** [comment]
but why we have 86 commits ...

**Pull request merged**